src/System.IO.Compression.Brotli/README.md

```diff
@@ -29,43 +29,41 @@ public static class Brotli {
 ```
 ```out bytesConsumed``` - number of bytes used from source data while executing the operation
 
-```out bytesWritten``` - number of bytes are written in destination while executing the operation
+```out bytesWritten``` - number of bytes that are written in destination while executing the operation
 
-```State``` is used in both compress and decompress mode to save a temporary status and data of process.  State will set automatically on initial call of Compress or Decompress. The state cannot be re-used for multiple data streams and cannot switch modes (compress vs. decompress).
+```State``` is used in both compress and decompress mode to save a temporary status and data of process.  State will be set automatically on initial call of Compress or Decompress. The state cannot be re-used for multiple data streams and cannot switch modes (compress vs. decompress).
 
-The ```Compress``` performs data compression. Taking data from ```source``` and writing compressed data to ```destination```.
+The ```Compress``` performs data compression. It takes data from ```source``` and writes compressed data to ```destination```.
 
 The ```FlushEncoder``` returns compressed data, which was sent to ```state``` using Compress. This method should always be called after all Compress executions (```isFinished = true```) or when you want to get compressed data immediately (```isFinished = false```).
 
-The ```Decompress``` Decompresses the data from ```source``` into ```destination```. 
+The ```Decompress``` decompresses the data from ```source``` into ```destination```. 
 
-```SetQuality``` allows you to set quality of compression ```0 to 11```. The higher quality means the higher compression ratio, but more compute time.
+```SetQuality``` allows you to set quality of compression ```between 0 to 11```. The higher quality means the higher compression ratio, but more compute time.
```

Comment:
"allows you to set the quality of compression between 0 and 11"
I would also add "inclusive" to be less ambiguous. Between 0 to 11 could mean [0, 11] or (0, 11).